src/cpp/server/server_cc.cc

```diff
@@ -607,7 +612,12 @@ void Server::PerformOpsOnCall(CallOpSetInterface* ops, Call* call) {
   grpc_op cops[MAX_OPS];
   ops->FillOps(call->call(), cops, &nops);
   auto result = grpc_call_start_batch(call->call(), cops, nops, ops, nullptr);
-  GPR_ASSERT(GRPC_CALL_OK == result);
+  if (result != GRPC_CALL_OK) {
+    gpr_log(GPR_ERROR, "Fatal: grpc_call_start_batch returned %d", result);
+    grpc_call_log_batch(const_cast<char*>(this_file), __LINE__,
```

Comment:
I assume that there's some reason we can't just inline `__FILE__` here?

Also, what about using the `GRPC_CALL_LOG_BATCH()` macro?

Reply:
Done. I changed the interface of `grpc_call_log_batch` from taking a `char *` to a `const char *`. 

The macro checks api tracer, while here I want to always generate the log.